src/RawData/RawDataController.ts

```diff
@@ -35,49 +39,56 @@ const TEMP_USER = "temp";
 
 const router = Router();
 
-// Can't use validator as multer uses form data to submit files
-router.post("/", upload.any(), async (req: Request, res: Response) => {
-  if (!req.files || !req.body.title) {
-    res.status(400).send({
-      message: "Missing file or title parameters"
+router.post(
+  "/file-ids",
+  validateBody(RawDataFileIdsValidator),
+  (req: TypedRequestBody<RawDataFileIds>, res: Response) => {
+    const fileIds = new Array(req.body.fileCount)
+      .fill(undefined)
+      .map(() => uuidv4());
+
+    res.status(200).send({
+      fileIds
     });
-    return;
   }
+);
 
-  const uploadId = uuidv4();
-  const collectionId = uuidv4();
-  const fileIds: string[] | undefined = Array.prototype.map.call(
-    req?.files,
-    (file: Express.Multer.File) => file.filename.split(".")[0]
-  );
-
-  // Insert plot collection and upload
-  await databaseConnection.query(INSERT_UPLOAD, [
-    uploadId,
-    TEMP_USER,
-    toDBDate(new Date()),
-    collectionId,
-    req.body.title,
-    req.body.description
-  ]);
-
-  // Insert file pointers simultaneously
-  const fileInserts = fileIds?.map((fileId) =>
-    databaseConnection.query(INSERT_FILE, [fileId, uploadId, collectionId])
-  );
-  await Promise.all(fileInserts);
-
-  res.status(200).send({ id: collectionId, fileIds });
+router.post("/chunk", upload.single("chunk"), (req: Request, res: Response) => {
+  res.status(200).send({ message: "Chunk succesfully uploaded" });
```

Comment:
```suggestion
  res.status(200).send({ message: `Chunk ${req.body.chunkCount} succesfully uploaded });
```
Make the return message a little more specific to each chunk, for logging purposes.

It may also be worth including the `fileId` in the message, something like "Chunk `x` for file `y` successfully uploaded"

Reply:
Done